pkg/crc/api/api.go

```diff
@@ -10,11 +10,11 @@ import (
 )
 
 func CreateApiServer(socketPath string) (CrcApiServer, error) {
-	listener, err := net.Listen("unix", socketPath)
+	listener, err := getListener(socketPath)
```

Comment:
This can call `createIPCListener()` directly now since `getListener()` is just `return createIPCListener()`